include/mesh/mesh_function.h

```diff
@@ -123,6 +123,15 @@ class MeshFunction : public FunctionBase<Number>,
                      const Real time=0.) libmesh_override;
 
   /**
+   * @returns a map of values of variable 0 at point
+   * \p p and for \p time, which defaults to zero.
+   * map is from element on Number and accounts for double defined
+   * values on faces when discontinuous variables are used
+   */
+  std::map<const Elem *, Number> discontinuousValue (const Point & p,
```

Comment:
Also, the comment no longer makes sense I think, as the default value is not zero but rather an empty map in this case?